ocaml/xapi/cert_distrib.mli

```diff
@@ -36,3 +25,28 @@ val import_joining_pool_certs :
     This parameter must be a result of [exchange_certificates_with_joiner].
     This functions was designed as part of pool join and is unlikely to be
     useful elsewhere. *)
+
+val collect_ca_certs :
+  __context:Context.t -> names:string list -> (string * string) list
+(** [collect_ca_certs ~__context ~names] returns the ca certificates present
```

Comment:
This type is mandated by the API design in idl: 
```ocaml
(* This is a map of uuid -> cert_blob *)
let certs = Map (String, String)
```

Right now it can be thought as an opaque token that gets serialized and usnerialized internally by the operations (in Wireprotocol exclusively) to remove as much danger as possible in this operations.

I don't know of a better type in the idl that wouldn't introduce much friction.